hadoop-hdds/common/dev-support/findbugsExcludeFile.xml

```diff
@@ -31,6 +31,10 @@
     <Method name="update" />
     <Bug pattern="SF_SWITCH_FALLTHROUGH,SF_SWITCH_NO_DEFAULT" />
   </Match>
+  <Match>
+    <Class name="org.apache.hadoop.hdds.utils.db.CodecBufferr" />
+    <Bug pattern="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE" />
+  </Match>
```

Comment:
Please merge from `master`, then this suppression can be removed:

```suggestion
```

Reply:
Merge master and the false positive did not reappear.